packages/ui-library/src/theme/variants/button-variants.ts

```diff
@@ -12,6 +12,10 @@ export const buildButtonVariant = (mode: 'light' | 'dark'): Components => ({
       endIcon: {
         marginLeft: `${DEFAULT_SPACING}px`,
       },
+      sizeLarge: {
+        fontSize: '1rem',
```

Comment:
how did you calculate this?

Reply:
Just saw in the design this should be button text styles for large case